seed/lib/superperms/orgs/migrations/0031_ecrypt_existing_audit_template_passwords

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.19 on 2023-05-11 18:49
+from django.db import migrations, transaction
+from seed.utils.encrypt import decrypt, encrypt
+
+@transaction.atomic
+def encrypt_existing_audit_template_passwords(apps, schema_editor):
+    Organization = apps.get_model('orgs', 'Organization')
+
+    for org in Organization.objects.iterator():
+        _, error = decrypt(org.audit_template_password)
```

Comment:
Is there any possibility that this field will already be encrypted before this migration?

Reply:
no I don't think so. there has not been an encryption mechanism prior to this PR.